packages/common/src/index.ts

```diff
@@ -8,6 +8,7 @@ export {
   KBarSearch,
   KBarResults,
   useRegisterActions,
+  useMatches,
```

Comment:
new kbar stuff.. didn't spend much time on this - the whole thing needs a bit more attention!

Reply:
ah yes. I had a look a while back and started to update the deprecated calls. got a bit messy, gave up, forgot about it. oops.